extensions/amp-apester-media/0.1/amp-apester-media.js

```diff
@@ -206,7 +206,11 @@ class AmpApesterMedia extends AMP.BaseElement {
           const height = 0 || media.data.size.height;
           if (height != this.height_) {
             this.height_ = height;
-            this./*OK*/attemptChangeHeight(height);
+            if (this.random_) {
+              this./*OK*/attemptChangeHeight(height);
+            } else {
+              this./*OK*/changeHeight(height);
```

Comment:
Correct. This is in response to Apester calculating the right height when embed block is generated.